amundsen_application/static/js/features/NavBar/index.tsx

```diff
@@ -100,7 +99,7 @@ export class NavBar extends React.Component<NavBarProps> {
                     alt=""
                   />
                 )}
-                <span className="title-3">{LOGO_TITLE}</span>
+                <span className="title-3">{AppConfig.logoTitle}</span>
```

Comment:
We want to move into a place where we never use the Configuration information directly like this, instead, we want to use tested getter functions from https://github.com/amundsen-io/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/config/config-utils.ts

Could you add two functions (getLogoTitle and getDocumentTitle) and their tests and use them in the code instead?

Thanks!